enclave_build/src/lib.rs

```diff
@@ -59,12 +60,12 @@ pub enum EnclaveBuildError {
     SerdeError(serde_json::Error),
     #[error("Error while parsing credentials: `{0}`")]
     CredentialsError(String),
-    #[error("Image cache initialization error: `{0:?}`")]
-    CacheInitError(std::io::Error),
-    #[error("Cache store operation failed: `{0:?}`")]
-    CacheStoreError(std::io::Error),
-    #[error("Cache entry not found or has wrong: `{0:?}`")]
-    CacheMissError(String),
+    #[error("Image storage initialization error: `{0:?}`")]
+    OciStorageInitError(std::io::Error),
```

Comment:
I've just noticed, you don't have to include `Error` in the variant name, it is clear that it's an error because it is part of an `Error` enum. Even if you only add other variants here, let's not propagate the issue, and name the new ones (or the changed ones) without the `Error` part.

Reply:
I can remove `Error` for the ones that express a problem like `OciStorageNotFound` or `OciStorageMalformed`. The others feel weird without naming them failures or errors. This should be consistent with the naming throughout the repo.

Reply:
Hmm I don't necessarily find it weird, for example if you have `Error::OciStorageInit` you would read this as: we have an error in the oci storage initialization. This is one of the consequences of [namespacing](https://en.wikipedia.org/wiki/Namespace), i.e. the `Error` part is simply redundant. If the name sounds weird maybe it is missing some verb, but the Error part should not be the one that is missing.

Reply:
Alright, makes sense. Changed the new ones.